public/mockServiceWorker.js

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable */
+/* tslint:disable */
+
+/**
+ * Mock Service Worker (1.2.1).
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
+ */
+
+const INTEGRITY_CHECKSUM = '3d6b9f06410d179a7f7404d4bf4c3c70'
```

Comment:
Is it ok to keep this unique ID this way?

Reply:
this file is generated by `msw` when we run the command `msw-init`, i don't think it has anything to do with privacy as `msw` will only be used for development and testing purposes

Reply:
update, we are also removing the file now, after the build has been created.